assets/components/payPalContributionButton/payPalContributionButton.jsx

```diff
@@ -52,7 +53,7 @@ const PayPalContributionButton = (props: PropTypes) =>
   (
     <button
       id="qa-contribute-paypal-button"
-      className="component-paypal-contribution-button"
+      className={`component-paypal-contribution-button ${props.additionalClass || ''}`}
```

Comment:
Is it possible to use the helper defined [here](https://github.com/guardian/support-frontend/blob/master/assets/helpers/utilities.js#L22) to handle this case?